package.json

```diff
@@ -60,18 +61,18 @@
     "parser": "babel-eslint"
   },
   "scripts": {
-    "lint": "node node_modules/standard/bin/cmd.js src/**/*.js",
```

Comment:
Why did you change these commands? Now users will have to install these 5 libraries globally in order to run the build. I'd prefer devs be able to just clone, `npm install` and `npm run all` without any additional setup.